db/structure.sql

```diff
@@ -2145,6 +2146,8 @@ INSERT INTO "schema_migrations" (version) VALUES
 ('20230306054610'),
 ('20231225134459'),
 ('20231225144737'),
-('20231231091716');
+('20231231091716'),
+('20241125123456'),
```

Comment:
The migration `20241125123456` is absent.

Reply:
Corrected.